tests/torch/conftest.py

```diff
@@ -48,6 +48,10 @@ def pytest_addoption(parser):
     parser.addoption(
         "--weekly-models", type=str, default=None, help="Path to models' weights for weekly tests"
     )
+    parser.addoption(
+        "--weekly-with-mixed-precision", action="store_true", default=False, help="Enable mixed precision for the"
```

Comment:
Not sure, it's a good way to extend weekly/nightly tests. It could lead to handling many options that means almost the same.
Ideally, there should be a single pytest marker: `weekly_tests` and `nightly_tests` similar to that: 
`
@pytest.mark.webtest
`
https://docs.pytest.org/en/7.1.x/example/markers.html#mark-examples

Reply:
I changed it, now you can run weekly test by
```python -m pytest -s tests/torch -m weekly  --weekly-models ...```
as well as nighlty test by
```python -m pytest -s tests/torch -m nightly ...```

Mixed precision for test sota checkpoint is not enabled by this PR. In case we need mixed precision for nightly test let's create a task for that

Reply:
Please see 96744